greedy.md

```diff
@@ -0,0 +1,15 @@
+Greedy Algorithms
+=================
+Here are some problems can be solved using a greedy strategy. A greedy approach
```

Comment:
I would suggest adding a clear definition of what Greedy algorithms entail. Maybe we can consider linking to a website that would further explain the topic?

Reply:
Or should we leave explaining the paradigm during progteam meetings?

Reply:
Yeah, I agree that It could use a little bit more explanation here.

I updated the intro, what do you think? @FernandoCaudillo10

Reply:
@donaldong I think this is great!